modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase

```diff
@@ -25,6 +25,51 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LRQA-69712. Verify that the user can approve a new account by Notifications. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanApproveByDetails {
+
+		// TODO LRQA-69712 CanApproveByDetails pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69708. Verify that the user can delete a new account. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanDeleteUser {
```

Comment:
Hi Kristin! Could you, please, change the test description to "Verify that the user can delete an account when there is a Workflow assigned"? Thank you! :)

Reply:
Updated the description for the test!